From 026bc08c36bdd4b1d19926771d55dcd2024ae46a Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Sun, 16 Jul 2017 13:38:51 +0300
Subject: [PATCH 1070/1345] dts: marvell: vanilla: Disable ppv2 10G ports on
 A8K&A7K DB boards

commit  6b17b2375802d58087246ab958d0ca1019ddb14d from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This patch disable ppv2 10G ports on DB board.
Current vanilla ppv2 driver didn't support GoP link interrupts
required for 10G ports support on DB board.

Change-Id: Ia33c8bfb9884601a8ce2946f2a8b69b1bf61c384
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/41625
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/marvell/armada-7040-db-pp2-vanilla.dts     |    2 +-
 .../dts/marvell/armada-8040-db-pp2-vanilla.dts     |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
index 1cb3bfe..3c5de47 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
@@ -146,7 +146,7 @@
 					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>;
 					port-id = <0>; /* pp2_port_id */
 					gop-port-id = <0>;
-					status = "okay";
+					status = "disabled";
 				};
 				eth1: eth1@020000 {
 					phy-mode = "sgmii"; /* lane-0 */
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
index a886b27..6ba5643 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
@@ -127,7 +127,7 @@
 					port-id = <0>; /* pp2_port_id */
 					gop-port-id = <0>;
 					phy-mode = "sfi";
-					status = "okay";
+					status = "disabled";
 				};
 				eth1: eth1@020000 {
 					status = "disabled";
@@ -198,7 +198,7 @@
 					port-id = <0>; /* pp2_port_id */
 					gop-port-id = <0>;
 					phy-mode = "sfi";
-					status = "okay";
+					status = "disabled";
 				};
 				eth1_1: eth1@020000 {
 					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>;
-- 
1.7.9.5

